src/_posts/2020-04-28-terraform-with-multiple-environments.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+asset-type: post
+name: terraform-with-multiple-environments
+title: Terraform With Multiple Environments
+date: 2020-04-22 10:00:00 +00:00
+author: Jorge Gueorguiev Garcia
+description: I have used a couple of different approaches to support multiple environments for Terraform deployments. We will investigate them here, and look at a third option.
+
+image:
+    src: /assets/custom/img/blog/railway.png
+    attribution:
+        text: Public domain photo from Pixabay
+        href: https://pixabay.com/photos/rails-soft-gleise-railway-3309912/
+tags:
+    - infrastructure
+    - devops
+    - Terraform
```

Comment:
Is this the Jeykll issue that's breaking the build here?
https://app.circleci.com/pipelines/github/codurance/site/60/workflows/52c2d479-67ea-43ec-aade-d243604e8cf7/jobs/6603

Reply:
The "Terraform" tag seems to be conflicting with an existing "terraform" tag

Reply:
Yep, that was the issue. I had to change it from terraform to Terraform, as there is another post with Terraform already

Reply:
Perhaps we should change the existing ones to 'terraform' for consistency with other tags?

Reply:
Was planning to do that during the weekend

Reply:
I'm happy to do it now. I'm on the repo anyway 

Done https://github.com/codurance/site/pull/1704/commits/0512ff7f6a2166190bce99d9a1ef460a2ce54daa